Compilation error, gcc 10.2

Archlinux GCC 10.2:
RecastDuring demo compliation:

```
../../Source/TestCase.cpp: In member function ‘bool TestCase::load(const string&)’:
../../Source/TestCase.cpp:158:32: error: ‘void* memset(void*, int, size_t)’ clearing an object of type ‘struct TestCase::Test’ with no trivial copy-assignment; use value-initialization instead [-Werror=class-memaccess]
  158 |    memset(test, 0, sizeof(Test));
      |                                ^
In file included from ../../Source/TestCase.cpp:23:
../../Include/TestCase.h:33:9: note: ‘struct TestCase::Test’ declared here
   33 |  struct Test
      |         ^~~~
../../Source/TestCase.cpp:172:32: error: ‘void* memset(void*, int, size_t)’ clearing an object of type ‘struct TestCase::Test’ with no trivial copy-assignment; use value-initialization instead [-Werror=class-memaccess]
  172 |    memset(test, 0, sizeof(Test));
      |                                ^
In file included from ../../Source/TestCase.cpp:23:
../../Include/TestCase.h:33:9: note: ‘struct TestCase::Test’ declared here
   33 |  struct Test
      |         ^~~~
cc1plus: all warnings being treated as errors
```

EDIT: Because I'm an absolute moron when it comes to C++ and i simply deleted the offending `memset` lines and thing worked.

--- a/RecastDemo/Include/TestCase.h
+++ b/RecastDemo/Include/TestCase.h
@@ -34,6 +34,10 @@ class TestCase
 	{
 		Test() :
 			type(),
+			spos(),
+			epos(),
+			nspos(),
+			nepos(),
 			radius(0),
 			includeFlags(0),
 			excludeFlags(0),

--- a/RecastDemo/Source/TestCase.cpp
+++ b/RecastDemo/Source/TestCase.cpp
@@ -154,8 +154,7 @@ bool TestCase::load(const std::string& filePath)
 		else if (row[0] == 'p' && row[1] == 'f')
 		{
 			// Pathfind test.
-			Test* test = new Test;
-			memset(test, 0, sizeof(Test));
+			Test* test = new Test();
 			test->type = TEST_PATHFIND;
 			test->expand = false;
 			test->next = m_tests;
@@ -168,8 +167,7 @@ bool TestCase::load(const std::string& filePath)
 		else if (row[0] == 'r' && row[1] == 'c')
 		{
 			// Pathfind test.
-			Test* test = new Test;
-			memset(test, 0, sizeof(Test));
+			Test* test = new Test();
 			test->type = TEST_RAYCAST;
 			test->expand = false;
 			test->next = m_tests;

